src/fklearn/metrics/pd_extractors.py

```diff
@@ -24,7 +26,27 @@ def split_evaluator_extractor_iteration(split_value, result, split_col, base_ext
 def split_evaluator_extractor(result, split_col, split_values, base_extractor):
     return pd.concat(
         list(map(split_evaluator_extractor_iteration(result=result, split_col=split_col, base_extractor=base_extractor),
-             split_values)))
+                 split_values)))
+
+
+@curry
+def temporal_split_evaluator_extractor(result, time_col, base_extractor, time_format="%Y-%m", eval_name=None):
+    if eval_name is None:
+        eval_name = 'split_evaluator__' + time_col
+
+    split_keys = [key for key in result.keys() if eval_name in key]
+    split_values = []
+    for key in split_keys:
+        date = key.split(eval_name)[1][1:]
+        try:
+            # just check time format
+            datetime.strptime(date, time_format)
+            split_values.append(date)
+        except ValueError:
+            # this might happen if result has temporal splitters using different data formats
+            pass
```

Comment:
Shall we maybe raise a warn?

Reply:
I don't think so. Let's say you have 2 time split evaluators, one weekly and the other daily. It's a "normal" use case. This extractor would extract only the one which matches the given `timeformat`